packages/client/src/types/Account.ts

```diff
@@ -26,6 +26,7 @@ export default interface Account {
   onboarded: boolean;
   customerId?: string;
   emailProvider?: string;
+  smsProvider?: string;
```

Comment:
it looks like they does not support multiple sms provider so we need to done that

Reply:
this is apply to remove smsAccountSid, smsAuthToken, smsFrom and replace that in every place in the system